services/web/server/src/simcore_service_webserver/projects/projects_db.py

```diff
@@ -463,10 +465,10 @@ async def _get_project(
             SELECT *
             FROM projects
             WHERE
-            {"" if include_templates else "projects.type != 'TEMPLATE' AND"}
+            {"projects.type = 'TEMPLATE' AND" if only_templates else ""}
             uuid = '{project_uuid}'
             AND (jsonb_exists_any(projects.access_rights, {_assemble_array_groups(user_groups)})
-            OR prj_owner = {user_id})
+            OR prj_owner = {user_id} {"OR published='true'" if only_published else ""})
```

Comment:
uff, this text expression is getting pretty. I guess at some point we should create the query with sqlalchemy query language?

Reply:
it's not scarry, that's just SQL. but I used chatgpt to convert it now to sqlalchemy. let's see if that works well